### PR TITLE
Merge up to date branch to master & review recording user activity changes

### PR DESCRIPTION
**Pull request by Zainrax**
Added ability to record user activity, and added some recording functionality to better suit the "stop starting" recording.

### CODE REVIEW

File: frontend/src/App.vue [hunk 1]
```diff
@@ -273,13 +287,69 @@ export default class App extends Vue {
...
+      state === ScreeningState.FRONTAL_LOCK ||
+      state === ScreeningState.STABLE_LOCK ||
+      state === ScreeningState.MULTIPLE_HEADS ||
+      state === ScreeningState.MEASURED ||
+      state === ScreeningState.TOO_FAR
+    );
+  }
+
+  private hasPersonExitFrame(frame: Frame, isInFrame: boolean): boolean {
+    const ThresholdSeconds = secondsToMiliseconds(4);
```
Comment (hardiesoft) [inline frontend/src/App.vue]: Small nit - `Miliseconds` -> `Milliseconds`

File: frontend/src/App.vue [hunk 2]
```diff
@@ -394,9 +468,16 @@ export default class App extends Vue {
     // Update the AppState:
     if (this.useLiveCamera) {
       this.appState.uuid = new Date().getTime();
+      this.isRecording = await DeviceApi.recorderStatus().then(
+        ({ recording }) => recording
+      );
+      DeviceApi.recordUserActivity =
+        window.localStorage.getItem("recordUserActivity") === "true"
```
Comment (hardiesoft) [inline frontend/src/App.vue:420]: Having this as a ternary expression seems a little redundant?

**Review by hardiesoft — APPROVED** [review]
It all looks pretty sane to me, only had a couple of tiny style comments, great work!

**Comment by Zainrax** [comment]
Great, will make those changes in next commit.

**Pull request merged**